modules/ding_user_frontend/ding_user_frontend.module

```diff
@@ -329,6 +329,17 @@ function ding_user_frontend_build_menu($path = NULL) {
         }
       }
     }
+
+    // Add logout to menu.
+    $logout_item = array();
+    $logout_item = $menu_item_defaults;
+    $logout_item['#title'] = t('Log out');
+    $logout_item['#href'] = '/user/logout';
```

Comment:
This should probably use [`url()`](https://api.drupal.org/api/drupal/includes%21common.inc/function/url/7.x) unless there is a specific reason to do otherwise.